doc/misc/npm-registry.md

```diff
@@ -21,6 +21,21 @@ The registry URL used is determined by the scope of the package (see
 supplied by the `registry` config parameter.  See `npm-config(1)`,
 `npmrc(5)`, and `npm-config(7)` for more on managing npm's configuration.
 
+## Does npm send any information about me back to the registry?
+
+When making requests of the registry npm adds two headers with information
+about your environment:
+
+* `Npm-Scope` – If your project is scoped, this header will contain its
+  scope. In the future npm hopes to build registry features that use this
+  information to allow you to customize your experience for your
+  organization.
+* `Npm-In-CI` – Set to "true" if npm believes this install is running in a
+  continous integration environment, "false" otherwise. This is detected by
+  looking for the following environment variables: `CI`, `TDDIUM`,
```

Comment:
maybe link to the code that does this? i imagine people will want to argue technically about how this is incomplete and/or not accurate so it may be helpful to tell them where a PR would be useful.